taxonomies/categories.txt

```diff
@@ -25007,21 +25154,29 @@ wikidata:en:Q746471
 en:French blue-veined cheeses
 de:Französischer Blauschimmelkäse
 fi:ranskalaiset sinihomejuustot
-fr:Fromages à pâte persillée français, pâtes à moisissure interne français, bleus français
+fr:Fromages à pâte persillée français, pâtes à moisissure interne français, bleus français, Fromage bleu au lait de vache
 nl:Franse blauwschimmelkazen
+ciqual_food_code:en:12520
+ciqual_food_name:en:Blue cheese, from cow's milk
 
 <en:French blue-veined cheeses
 <en:Cow cheeses
 fr:Bleus d'Auvergne
+en:Auvergne blue cheese from cow's milk
 de:Bleu d'Auvergne
 nl:Bleus d'Auvergne
 wikidata:en:Q883954
 country:en:France
+ciqual_food_code:en:12521
+ciqual_food_name:en:Auvergne blue cheese, from cow's milk
 
 <en:French blue-veined cheeses
 fr:Bleus de Gex, bleus du Haut-Jura, bleus de Septmoncel
```

Comment:
We should make 3 categories, one for each cheeese.  Also if there is an en: name, it needs to be first, before the fr: name